arm-compute/quickstart-templates/aliases.json

```diff
@@ -0,0 +1,83 @@
+{
+    "$schema":"http://schema.management.azure.com/schemas/2015-01-01-preview/deploymentTemplate.json",
+    "contentVersion": "1.0.0.0",
+    "parameters": {},
+    "variables": {},
+    "resources": [],
+    "outputs": {
+	"aliases": {
+	    "type": "object",
+	    "value": {
+		"Linux": {
+		    "CentOS":{
+			"publisher":"OpenLogic",
+			"offer":"CentOS",
+			"sku":"7.2",
+			"version":"latest"
+		    },
+		    "CoreOS":{
+			"publisher":"CoreOS",
+			"offer":"CoreOS",
+			"sku":"Stable",
+			"version":"latest"
+		    },
+		    "Debian":{
+			"publisher":"credativ",
+			"offer":"Debian",
+			"sku":"8",
+			"version":"latest"
+		    },
+		    "OpenSUSE":{
+			"publisher":"SUSE",
+			"offer":"openSUSE",
+			"sku":"13.2",
+			"version":"latest"
+		    },
+		    "RHEL":{
+			"publisher":"RedHat",
+			"offer":"RHEL",
+			"sku":"7.2",
+			"version":"latest"
+		    },
+		    "SLES":{
+			"publisher":"SUSE",
+			"offer":"SLES",
+			"sku":"12-SP1",
+			"version":"latest"
+		    },
+		    "UbuntuLTS":{
+			"publisher":"Canonical",
+			"offer":"UbuntuServer",
+			"sku":"14.04.4-LTS",
+			"version":"latest"
+		    }
+		},
+		
+		"Windows": {
+		    "Win2012R2Datacenter": {
+			"publisher":"MicrosoftWindowsServer",
+			"offer":"WindowsServer",
+			"sku":"2012-R2-Datacenter",
+			"version":"latest"
+		    },
+		    "Win2012Datacenter": {
+			"publisher":"MicrosoftWindowsServer",
+			"offer":"WindowsServer",
+			"sku":"2012-Datacenter",
+			"version":"latest"
+		    },
+		    "Win2008R2SP1": {
+			"publisher":"MicrosoftWindowsServer",
+			"offer":"WindowsServer",
+			"sku":"2008-R2-SP1",
+			"version":"latest"
+		    }
+		}
+	    },
+
+	    "metadata": {
+		"description": "This list of aliases is used by CLI, Powershell, and Portal as shorthands for commonly used VM images. If you change this file, please verify that this doesn't break VMSS creation from Portal :)."
```

Comment:
What do you mean by "both VM and VMSS"? If you're referring to the description, then for the time being there is no VM creation from portal using these aliases